mlflow/models/evaluation/default_evaluator.py

```diff
@@ -116,17 +116,28 @@ def _extract_predict_fn(model, raw_model):
     return predict_fn, predict_proba_fn
 
 
-def _get_regressor_metrics(y, y_pred):
+def _get_regressor_metrics(y, y_pred, sample_weights):
+    sum_on_label = (
+        sum(np.array(y) * np.array(sample_weights)) if sample_weights is not None else sum(y)
```

Comment:
```suggestion
       (np.array(y) * np.array(sample_weights)).sum() if sample_weights is not None else sum(y)
```

Can we use `numpy.sum` here?


#### `sum` vs `numpy.sum`


```python
import numpy as np
import timeit

x = np.array(range(1_000_000))


def builtin_sum():
    return sum(x)


def numpy_sum():
    return x.sum()


assert builtin_sum() == numpy_sum()

print(timeit.timeit(builtin_sum, number=100))
print(timeit.timeit(numpy_sum, number=100))
```

```
3.738885705002758
0.0338254450034583
```

Reply:
Great idea! Updated!